packages/itwinui-react/src/core/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -57,8 +59,9 @@ type ToggleSwitchProps = {
  *
  * ---
  *
- * `className` and `style` props are applied depending on `wrapperProps`:
- * - `wrapperProps!=null`: `className` and `style` applied on the `input` element where
+ * `className` and `style` props are applied depending on `wrapperProps` and `ThemeProvider`'s
+ * `future.consistentPropsSpread` prop:
+ * - `wrapperProps!=null || consistentPropsSpread=true`: `className` and `style` applied on the `input` element where
```

Comment:
I find this style of documentation confusing because it uses fake code syntax instead of natural language. "`wrapperProps!=null || consistentPropsSpread=true`" is not real code.

It would be more clear to instead write as something like below:

> By default, `className` and `style` are applied on the wrapper element, and all other DOM props are passed to `<input>` element.
>
> When `wrapperProps` is passed or when the `ToggleSwitch.consistentPropsSpread` future flag is enabled, `className` and `style` are applied on the `<input>` element, similar to other DOM props.

Reply:
Considering that I personally found it confusing, I'd say the previous version was less clear and had more room for misinterpretation, especially since there is no indication which parts are real code and which parts are pseudocode.

It's best to keep the inline code parts (within single backticks) relatively simple so that the sentence is easier to read. This can of course be combined with dedicated code blocks for the best of both words (as your commit has correctly done).

My hope is that we can write better, more consistent documentation going forward. It has always been a weak point for iTwinUI.